pkg/commands/git_commands/rebase.go

```diff
@@ -228,7 +226,8 @@ func (self *RebaseCommands) AmendTo(commits []*models.Commit, commitIndex int) e
 	}
 
 	// Get the sha of the commit we just created
-	fixupSha, err := self.cmd.New("git rev-parse --verify HEAD").RunWithOutput()
+	cmdStr := NewGitCmd("rev-parse").Arg("--verify").Arg("HEAD").ToString()
```

Comment:
Could use `Args` here. There are several other places as well still left to be replaced with `Args`.

Reply:
Definiitely